apps/design-land/src/app/sidebar/sidebar.component.html

```diff
@@ -1,20 +1,18 @@
 <daff-form-field>
   <select daff-native-select [(ngModel)]="mode">
     <option value="push">Push (push)</option>
-    <option value="side">Side (side)</option>
     <option value="over">Over (over)</option>
-    <option value="fixed">Fixed (fixed)</option>
   </select>
 </daff-form-field>
 
 <button daff-button (click)="open = !open">{{ open ? "Close" : "Open" }}</button>
+<button daff-button color="secondary" (click)="fixed = !fixed">{{ fixed ? "Undo" : "Fix in Viewport" }}</button>
 
-<daff-sidebar-viewport [mode]="mode" [opened]="open">
-    <daff-sidebar>
-      <a daff-link-set-item routerLink="accordion">Accordion</a><br>
-      <a daff-link-set-item routerLink="article">Article</a><br>
-    </daff-sidebar>
-    <p>Some Content</p>
-</daff-sidebar-viewport>
-<h1>Outside Focus   </h1>
-<input>    <input>     
\ No newline at end of file
+
+<daff-sidebar-viewport [mode]="mode" [opened]="open" [fixed]="fixed">
+  <daff-sidebar>
+      <a routerLink="accordion">Accordion</a><br>
```

Comment:
Should be
```
<daff-link-set>
    <a daff-link-set-item routerLink="accordion">Accordion</a>
    <a daff-link-set-item routerLink="article">Article</a>
</daff-link-set>
```

Reply:
fixed in ca31103737ff13c2b0e15942fc3032ead77086da